18F/identity-idp
# LG-13196 Enhanced IPP Ready to Verify Email - Take Two

## 🎫 Ticket
* Pull Request to address comments in [PR# 10816](https://github.com/18F/identity-idp/pull/10816) that were made after the pull request had been merged into main
* Later also discovered (while testing on staging) that the images on my email did not display (even though they displayed locally on email template)

![Screenshot 2024-06-20 at 2 03 48 PM](https://github.com/18F/identity-idp/assets/125507397/2aacd35e-b277-45cf-a88a-d4132789cdd7)


Original ticket: [LG-13196](https://cm-jira.usa.gov/browse/LG-13196) Implementation: EIPP "You're ready to verify in person" Barcode email

## 🛠 Summary of changes
- Delete deprecated width attribute on td and replaced it with CSS style="width:xxpx" to achieve the same layout 
 (I had not previously modified reset_password_instructions but it used the same deprecated attribute so I fixed it.)
- Round down on image width where a decimal was used
- Used USWDS class 
- For each SVG image in the mailer- I make a png version. The Ready to Verify mailer is now using the png image. The SVG images did not get deleted because they are still being used on the Ready to Verify view. 

## 📜 Testing Plan

1. Create an EIPP Enrollment by starting up Sinatra.
2. Inspect the Ready To Verify Email Template (EIPP Version). Layout/style should not have changed and should match [mocks](https://www.figma.com/design/Qu8duCyL9wa4JpL5gqSFkT/LG-12660-LG-12988-EIPP-Design-%2F-Content-EIPP-workflow-for-F%26F-pilot?node-id=1419-23102&m=dev).  Confirm Ready to Verify is not affected just because it is just one more click.
3. If you prefer not to run through the entire EIPP flow you can jump to rails/mailer view once you get the sever going:
- [ ] http://localhost:3000/rails/mailers/user_mailer/in_person_ready_to_verify
- [ ] http://localhost:3000/rails/mailers/user_mailer/in_person_ready_to_verify_enhanced_ipp_enabled
- [ ] http://localhost:3000/rails/mailers/user_mailer/reset_password_instructions 
4. Once this is merged into staging- run through and create an EIPP enrollment. Open the email from staging that got delivered and confirm you can see the icons

## 👀 Screenshots

![Screenshot 2024-06-21 at 9 19 31 AM](https://github.com/18F/identity-idp/assets/125507397/ba769f0a-d38d-4e00-9e15-09071bb45644)

![Screenshot 2024-06-21 at 9 19 48 AM](https://github.com/18F/identity-idp/assets/125507397/6faf4526-04cb-4f62-9317-427d99575dfa)

![Screenshot 2024-06-21 at 9 19 52 AM](https://github.com/18F/identity-idp/assets/125507397/b4b184ab-5528-4fdc-b417-304302c3b8ad)
